scripts/gha/restore_secrets.py

```diff
@@ -51,6 +53,7 @@
 flags.DEFINE_string("passphrase_file", None,
                     "Path to file with passphrase. Use \"-\" (without quotes) for stdin.")
 flags.DEFINE_string("artifact", None, "Artifact Path, google-services.json will be placed here.")
+flags.DEFINE_string("api", None, "Retrieve secret for the particular SDK only.")
```

Comment:
Good idea. And what about make it a list of api?
`flags.DEFINE_list("apis", None, "Retrieve secret for the particular SDKs only.")`

Reply:
Is there a usecase for this?  Generally we're only working on one API/SDK at a time, or all at the same time in the case of CI.

Reply:
e.g. When someone tries to build only the failed testapps locally.

Reply:
Done!  PTAL!